modules/ghlint-cli/src/main/kotlin/net/twisterrob/ghlint/cli/RuleHelpService.kt

```diff
@@ -0,0 +1,73 @@
+package net.twisterrob.ghlint.cli
+
+import net.twisterrob.ghlint.rule.Example
+import net.twisterrob.ghlint.rule.Issue
+import net.twisterrob.ghlint.rule.Rule
+import net.twisterrob.ghlint.rules.DefaultRuleSet
+
+internal class RuleHelpService {
+
+	private val allRules: List<Rule> by lazy {
+		DefaultRuleSet().createRules()
+	}
+
+	fun getRuleHelp(ruleId: String): String? {
+		val issue = findIssueById(ruleId) ?: return null
+		return formatIssueHelp(issue)
+	}
+
+	private fun findIssueById(ruleId: String): Issue? {
+		for (rule in allRules) {
+			rule.issues.forEach { issue ->
+				if (issue.id == ruleId) {
+					return issue
+				}
+			}
+		}
+		return null
+	}
+
+	private fun formatIssueHelp(issue: Issue): String {
```

Comment:
Consider using StringBuilder.buildString { } or a more functional approach with joinToString() for better Kotlin idioms and readability.